src/components/Picklist/index.js

```diff
@@ -159,6 +161,13 @@ class Picklist extends Component {
         }, 0);
     }
 
+    handleContainerClick(event) {
+        const { isOpen } = this.state;
+        if (isOpen) {
+            event.nativeEvent.stopImmediatePropagation();
```

Comment:
the issue with this solution is that if someone adds a listener at the document it will not let them get these click events